cpp/tests/bitmask/valid_ops_test.cu

```diff
@@ -53,6 +53,7 @@ TEST_F(ValidsTest, NullValids)
   gdf_error error_code = gdf_count_nonzero_mask(nullptr, size, &count);
 
   ASSERT_EQ(GDF_SUCCESS,error_code) << "GDF Operation did not complete successfully.";
+
```

Comment:
Suggest not adding a single space to the file...
```suggestion
```

Reply:
We need to get clang-format so that things like this quit clogging up reviews... :smile: